package.json

```
@@ -79,7 +79,7 @@
   "dependencies": {
     "@dotlottie/react-player": "^1.6.3",
     "@expensify/react-native-background-task": "file:./modules/background-task",
-    "@expensify/react-native-live-markdown": "0.1.230",
+    "@expensify/react-native-live-markdown": "^0.1.233",
```

```suggestion
    "@expensify/react-native-live-markdown": "0.1.233",
```
Please keep the version specific like it was before.